twake/frontend/src/app/scenes/Client/WorkspacesBar/Components/Group.js

```diff
@@ -88,7 +92,8 @@ export default class Group extends Component {
         text: Languages.t('scenes.app.workspacesbar.components.create_company_menu'),
         icon: 'plus',
         onClick: () => {
-          popupManager.open(<CreateCompanyView />);
+          //popupManager.open(<CreateCompanyView />);
+          console.log(Groups.user_groups);
```

Comment:
Is it expected this console.log + the fact that the button does nothing now?

Reply:
Definitly not excepted, back to previous state. Thank you :+1: